src/Microsoft.ML.Transforms/OneHotHashEncoding.cs

```diff
@@ -207,42 +207,42 @@ public sealed class OneHotHashEncodingEstimator : IEstimator<OneHotHashEncodingT
         [BestFriend]
         internal static class Defaults
         {
-            public const int HashBits = 16;
+            public const int NumberOfHashBits = 16;
             public const uint Seed = 314489979;
-            public const bool Ordered = true;
-            public const int InvertHash = 0;
-            public const OneHotEncodingTransformer.OutputKind OutputKind = OneHotEncodingTransformer.OutputKind.Bag;
+            public const bool UseOrderedHashing = true;
+            public const int MaximumNumberOfInverts = 0;
+            public const OneHotEncodingEstimator.OutputKind OutputKind = OneHotEncodingEstimator.OutputKind.Bag;
         }
 
         /// <summary>
         /// Describes how the transformer handles one column pair.
         /// </summary>
         public sealed class ColumnOptions
         {
-            public readonly HashingEstimator.ColumnOptions HashInfo;
-            public readonly OneHotEncodingTransformer.OutputKind OutputKind;
+            public readonly HashingEstimator.ColumnOptions HashingOptions;
+            public readonly OneHotEncodingEstimator.OutputKind OutputKind;
 
             /// <summary>
             /// Describes how the transformer handles one column pair.
             /// </summary>
             /// <param name="name">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
             /// <param name="inputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="name"/> will be used as source.</param>
             /// <param name="outputKind">Kind of output: bag, indicator vector etc.</param>
-            /// <param name="hashBits">Number of bits to hash into. Must be between 1 and 31, inclusive.</param>
+            /// <param name="numberOfHashBits">Number of bits to hash into. Must be between 1 and 31, inclusive.</param>
             /// <param name="seed">Hashing seed.</param>
-            /// <param name="ordered">Whether the position of each term should be included in the hash.</param>
-            /// <param name="invertHash">During hashing we constuct mappings between original values and the produced hash values.
+            /// <param name="useOrderedHashing">Whether the position of each term should be included in the hash.</param>
+            /// <param name="maximumNumberOfInverts">During hashing we constuct mappings between original values and the produced hash values.
             /// Text representation of original values are stored in the slot names of the  metadata for the new column.Hashing, as such, can map many initial values to one.
-            /// <paramref name="invertHash"/> specifies the upper bound of the number of distinct input values mapping to a hash that should be retained.
+            /// <paramref name="maximumNumberOfInverts"/> specifies the upper bound of the number of distinct input values mapping to a hash that should be retained.
```

Comment:
nit - > S in specifies should be capitalized.

Reply:
I think that's a `paramref` so we should be good!